python/simpler/mpi_l3_session.py

```
@@ -0,0 +1,610 @@
+                    MailboxRequestState.REQUEST_READY,
+                    MailboxRequestState.SHUTDOWN_READY,
+                ):
+                    if mailbox.group_state is MailboxGroupState.TERMINAL:
+                        break
+                if mailbox.group_state is MailboxGroupState.TERMINAL:
+                    request = None
+                else:
+                    try:
+                        request = mailbox.accept_request(last_sequence_id=last_sequence_id)
+                        last_sequence_id = request.sequence_id
+                    except BaseException:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)